lib/workers/global/index.ts

```diff
@@ -156,15 +156,18 @@ export async function start(): Promise<number> {
           );
         }
       }
-      // initialize all submodules
-      config = await globalInitialize(config);
 
       // Set platform and endpoint in case local presets are used
+      // Set allowedHeaders in case hostRules headers are configured in file config
       GlobalConfig.set({
```

Comment:
I think we need to set it twice, because endpoint can be changed from platform init. (eg strip api suffix...)